examples/wasm-in-web-worker/worker.js

```diff
@@ -11,7 +11,7 @@ const {NumberEval} = wasm_bindgen;
 
 async function init_wasm_in_worker() {
     // Load the wasm file by awaiting the Promise returned by `wasm_bindgen`.
-    await wasm_bindgen('./pkg/wasm_in_web_worker_bg.wasm');
+    await wasm_bindgen();
```

Comment:
Won't this not work due to #3277? I haven't actually tested this as I'm on my phone right now.

Reply:
Yep ...

Reply:
Fixed.
Thank you for reviewing all my PRs so attentively.